kepler/app.py

```diff
@@ -45,20 +46,6 @@ def handle_unsupported_format(error):
 
 
 def register_loggers(app):
-    formatter = logging.Formatter("[%(name)s] [%(levelname)s] %(message)s")
-
-    logging.getLogger().setLevel(logging.DEBUG)
-
-    # stdout log handler
-    handler = logging.StreamHandler()
-    handler.setFormatter(formatter)
-    handler.setLevel(logging.DEBUG)
-
-    # general flask app logger
-    app.logger.addHandler(handler)
-
-    # http logging
-    requests_log = logging.getLogger('requests.packages.urllib3')
-    requests_log.setLevel(logging.DEBUG)
-    requests_log.addHandler(handler)
+    with open('kepler/logging.conf') as f:
```

Comment:
We should pull the location of the configuration file from an environment variable. I'm fine with having it default to this location, but we may want to be able to use a different config for testing, for example.